src/network/requestQueue/index.js

```diff
@@ -42,6 +43,14 @@ module.exports = class RequestQueue {
           queueSize: this.pending.length,
         })
     }
+
+    this.requestTimeoutIntervalId = setInterval(() => {
```

Comment:
We should only do this is `enforceRequestTimeout` is enabled, otherwise there's no need to create the setInterval.